Bottom: d02b040c5844f45411d084681595e9da849b10c6
Top:    30fbaade5ac6ae1f1c3a73200a98f8e7e7d1b1f5
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-22 16:50:33 -0400

Refresh of i2s-cell-clock

---

diff --git a/sound/soc/fsl/mpc5200_psc_i2s.c b/sound/soc/fsl/mpc5200_psc_i2s.c
index b76426c..35dbb18 100644
--- a/sound/soc/fsl/mpc5200_psc_i2s.c
+++ b/sound/soc/fsl/mpc5200_psc_i2s.c
@@ -149,7 +149,7 @@ static int psc_i2s_set_sysclk(struct snd_soc_dai *cpu_dai,
 			dev_dbg(psc_dma->dev, "psc_i2s_set_sysclk(clkdiv %d freq error=%ldHz)\n",
 					clkdiv, (ppc_proc_freq / clkdiv - freq));
 
-			return mpc52xx_set_psc_clkdiv(psc_dma->dai[0].id + 1, clkdiv);
+			return mpc52xx_set_psc_clkdiv(psc_dma->id + 1, clkdiv);
 		}
 	}
 	return 0;
